### DISCUSSION TITLE
我主页一直显示empty啊

### DISCUSSION THREAD

**Discussion by mztxy229** [share121/inter-knot#4603] [灌水]
大概一天前我上课的时候摸鱼想上绳网看看，然后就显示api什么鬼的（各位应该也见过），之前就是等一会他就好了，但是这次他应该崩了小一天。。。？我一般用safari浏览器然后进去他就给我显示一个empty，啥帖子都没有。我本来刚研究明白那个米游社表情包脚本怎么下载想试试来着，他就，崩了。我现在用的chrome倒是没什么问题，所以到底咋了

**Comment by hk0852** [comment]
因为你的更新版本了，之前的版本已经被废弃掉了

**Comment by Yizakl** [comment]
api超限需要在设置换用户接口
